src/coreclr/scripts/superpmi.py

```diff
@@ -1872,8 +1889,8 @@ async def create_one_artifact(jit_path: str, location: str) -> str:
                                 return generated_txt
 
                             # Generate diff and base JIT dumps
-                            base_txt = await create_one_artifact(self.base_jit_path, base_location)
-                            diff_txt = await create_one_artifact(self.diff_jit_path, diff_location)
+                            base_txt = await create_one_artifact(self.base_jit_path, base_location, flags + base_option_flags)
+                            diff_txt = await create_one_artifact(self.diff_jit_path, diff_location, flags + diff_option_flags)
```

Comment:
This isn't going to work for creating the asm files for diffs, because it will pass `-jit2option` (when that is fixed), however, we're only passing a single JIT in the "create dasm file artifacts" case (as opposed to the "do asm diffs by passing two JITs and `-a`). So you need (with better names) `diff_option_flags_for_asmdiffs` using `-jit2option` and `diff_option_flags_for_generating_dasm` using `-jitoption`.

Reply:
Ah, ok. I only tried it for the base jit... should have tried it with the diff jit too.